javascript/pull-view.js

```diff
@@ -165,14 +170,44 @@
       } else {
         text = 'Unknown';
       }
-
       // if status is success but PR is not mergable, overwrite status...
       if (success && info.get('mergeable') === false){
         classes = 'not-mergeable';
         text = 'Merge Conflicts';
       }
-
       return '<span class="status ' + classes + '">' + text + '</span>';
+    },
+
+    generateApprovalHTML: function(info,reviews) {
+      var classes = '';
+      var text = '';
+      var reviewers = reviews.get('reviewers')
+
+      if (reviewers != null){
+          var totalReviews = Object.keys(reviewers).length;
+          var approved = 0;
+          var requestedChanges = 0;
+
+          for(var review in reviewers) {
+            if(reviewers[review].state == "APPROVED"){
+                approved ++;
```

Comment:
Some whitespace stuff here needs changing to match the style in the rest of the file